vignettes/ebola.Rmd

```diff
@@ -0,0 +1,615 @@
+```
+
+Stratification by sex-district:
+
+```{r}
+newdata_sex_district_pmf <- obs_prep |>
+  as.data.frame() |>
+  modelr::data_grid(sex, district) |>
+  dplyr::mutate(obs_t = 1000, pwindow_upr = 1, swindow_upr = 1)
+
+draws_sex_district_pmf <- predicted_draws(
+  fit_sex_district,
+  newdata = newdata_sex_district_pmf,
+  ndraws = 1000
+)
+
+draws_sex_district_pmf |>
+  dplyr::mutate(
+    sex = dplyr::case_when(
+      sex == "F" ~ "Female",
+      sex == "M" ~ "Male"
+    )
+  ) |>
+  ggplot(aes(x = .prediction)) +
+  geom_bar(aes(y = after_stat(count / sum(count)))) +
+  labs(x = "PMF with daily censoring and no truncation", y = "") +
+  facet_grid(district ~ sex) +
+  theme_minimal()
+```
+
+### Continuous probability density function
+
+The posterior predictive distribution under no truncation and no censoring.
+That is to produce continuous delay times.
+I don't like that `geom_density()` is doing kernel smoothing but plotting as a histogram is suggestive of there being discrete structure.
+
+No stratification:
+
+```{r}
+newdata_pdf <- obs_prep |>
+  as.data.frame() |>
+  dplyr::mutate(obs_t = 1000, pwindow_upr = 0.001, swindow_upr = 0.001)
+
+draws_pdf <- predicted_draws(fit, newdata = newdata_pdf, ndraws = 1000)
+
+ggplot(draws_pdf, aes(x = .prediction)) +
+  geom_density() +
+  labs(x = "PDF with no censoring and no truncation", y = "") +
+  theme_minimal()
+```
+
+Stratification by sex:
+
+```{r}
+newdata_sex_pdf <- obs_prep |>
+  as.data.frame() |>
+  modelr::data_grid(sex) |>
+  dplyr::mutate(obs_t = 1000, pwindow_upr = 0.001, swindow_upr = 0.001)
+
+draws_sex_pdf <- predicted_draws(
+  fit_sex,
+  newdata = newdata_sex_pdf,
+  ndraws = 1000
+)
+
+draws_sex_pmf |>
+  dplyr::mutate(
+    sex = dplyr::case_when(
+      sex == "F" ~ "Female",
+      sex == "M" ~ "Male"
+    )
+  ) |>
+  ggplot(aes(x = .prediction)) +
+  geom_density() +
+  labs(x = "PDF with no censoring and no truncation", y = "") +
+  facet_grid(. ~ sex) +
+  theme_minimal()
+```
+
+Stratification by sex:
+
+```{r}
+newdata_sex_district_pdf <- obs_prep |>
+  as.data.frame() |>
+  modelr::data_grid(sex, district) |>
+  dplyr::mutate(obs_t = 1000, pwindow_upr = 0.001, swindow_upr = 0.001)
+
+draws_sex_district_pdf <- predicted_draws(
+  fit_sex_district,
+  newdata = newdata_sex_district_pdf,
+  ndraws = 1000
+)
+
+draws_sex_district_pdf |>
+  dplyr::mutate(
+    sex = dplyr::case_when(
+      sex == "F" ~ "Female",
+      sex == "M" ~ "Male"
+    )
+  ) |>
+  ggplot(aes(x = .prediction)) +
+  geom_density() +
+  labs(x = "PDF with no censoring and no truncation", y = "") +
+  facet_grid(district ~ sex) +
+  theme_minimal()
+```
+
+## Model comparison
+
+* Use `loo` and PSIS
+* Find out if it's worth including sex as a covariate. Presumably yes given how different the delay distributions are
+* Find out if it's worth including district as a random effects
+  * What about spatially structured? Probably future work: how to include more structured random effects
```

Comment:
1000 million percent this is future work

Reply:
https://github.com/epinowcast/epidist/issues/285